lib/graphql/language/parser.y

```diff
@@ -372,6 +372,24 @@ def next_token
   end
 end
 
+def get_description(token)
+  comments = []
+
+  loop do
+    token = token.prev_token
+
+    break if token.nil?
+    break if token.name != :COMMENT
+    break if token.next_token.line != token.line + 1
+
+    comments << token.to_s[2..-1] || ""
+  end
+
+  return nil if comments.empty?
+
+  comments.reverse.join("\n")
```

Comment:
I guess since this is only for schemas, this isn't a super hot path, but if we want to avoid the intermediary array from `.reverse` later, I think we could use `.unshift`, eg

```ruby 
loop do 
  prev_token = token 
  token = token.prev_token
  if token && token.name == :COMMENT && prev_token.line == token.line + 1
    comments.unshift(token.to_s.sub(/^#\s*/, "")
  end 
end 

comments.join("\n")
```